### PR TITLE
Feature/registration tests

### PR DESCRIPTION
**Pull request by jpaidoussi**
Initial attempt at tests for registration migrations.

### CODE REVIEW

**Comment by jpaidoussi** [comment]
The diff for this PR is not great due to adding in the raw SQL dump files.

File: tests/test_migrations.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+@pytest.fixture(scope='module')
+def migrator(config, request):
+    # Create all the test DBs if they don't already exist.
+    for db_url in config.DATABASES.values():
+        if not database_exists(db_url):
+            create_database(db_url)
+            url = make_url(db_url)
+            import_sql_schema(request.fspath.dirname, url.database)
+    yield migrations.Migrator(config, echo=mock_echo)
+    # Drop the test DBs on teardown.
```
Comment (smn) [inline tests/test_migrations.py:48]: I may be missing something obvious here but what does the yield do?

Comment (jpaidoussi) [reply tests/test_migrations.py:48]: You aren't missing anything obvious - it is pytest's way of handling fixture teardowns (http://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code). It isn't my taste but I went for expedience.

Comment (smn) [reply tests/test_migrations.py:48]: :+1:

**Comment by smn** [comment]
:+1:

**Pull request merged**